src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] These action types need to follow the duck's pattern, according to the project requirements:

![image](https://user-images.githubusercontent.com/57421823/166410250-2c9ee9cc-0662-4cbf-9eec-b8e1eb9f89a0.png)

Hint:  Examples of duck's pattern would be: 

```javascript
const CHECK_STATUS = bookstore/categories/CHECK_STATUS
```

Reply:
@Meltrust  also added here 
![e60](https://user-images.githubusercontent.com/40554384/166630239-6a9ce9f3-51da-4d21-ad35-87586f41dd63.PNG)